maven-settings.xml

```diff
@@ -0,0 +1,46 @@
+<settings xmlns="http://maven.apache.org/SETTINGS/1.0.0"
```

Comment:
actually this is vestigial, since i added the required sections here for html api doc generation under `stabilization_settings.xml` so I will just remove this